.pipelines/build.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-2019'
+
+variables:
+  solution: 'JWT.sln'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+  dotNetVersion: '2.2.106'
+
+steps:
+- task: DotNetCoreInstaller@0
+  displayName: Install .NET Core v$(dotNetVersion)
+  inputs:
+      version: $(dotNetVersion)
+
+- task: MSBuild@1
```

Comment:
Wondering why do we have to use MSBuild over `dotnet build`?

Reply:
The build is green now. I think this the last open item before it's good to go in.

Reply:
You can go back to dotnet build. This was when I was trying to resolve the NuGet issues but I solved that by upgrading the project to using project references instead of packages.config

Reply:
Doesn't work. Maybe because we have here a mix of .NET Core and .NET Framework projects. I'll figure this out in another PR.